mmv1/products/vertexai/Models.yaml

```diff
@@ -0,0 +1,444 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: Models
+base_url: projects/{{project}}/locations/{{region}}/models
+create_url: 'projects/{{project}}/locations/{{region}}/modelsPRE_CREATE_REPLACE_ME'
+delete_url: '{{name}}'
+self_link: '{{name}}'
+
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/vertex-ai/docs'
+  api: 'https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.models'
+description: |-
+  A trained machine learning Model.
+id_format: '{{name}}'
+autogen_async: false
+exclude_import: true
+async:
+  actions:
+    - create
+    - delete
+  operation:
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result:
+    path: 'response'
+    resource_inside_response: true
+  status:
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error:
+    path: 'error'
+    message: 'message'
+custom_code:
+  decoder: templates/terraform/decoders/vertex_ai_models.go.erb
+  pre_create: templates/terraform/pre_create/vertex_ai_models.go.erb
+examples:
+  - name: 'vertex_ai_model_source_basic'
+    skip_import_test: true
```

Comment:
```suggestion
    exclude_import_test: true
```

Reply:
latest commit fixes the errors from running magic modules provider generation.